src/BinlogTool/SaveFiles.cs

```diff
@@ -103,16 +104,30 @@ private void WriteEmptyAssembly(string physicalReferencePath)
 
         private string TryGetReference(string argument)
         {
+            string reference;
             if (argument.StartsWith("/r:"))
             {
-                return argument.Substring(3);
+                reference = argument.Substring(3);
             }
             else if (argument.StartsWith("/reference:"))
             {
-                return argument.Substring(11);
+                reference = argument.Substring(11);
+            }
+            else
+            {
+                return null;
             }
 
-            return null;
+            if (reference.Length >= 2)
+            {
+                // Remove enclosing quotes if present
+                if ((reference.StartsWith('\'') && reference.EndsWith('\'')) ||

```

Comment:
I have a TrimQuotes() extension method in TextUtilities.cs, not sure if it's available in this project, we should probably just add a linked file if it's not already linked.